src/resources/develop/Currencies.yaml

```diff
@@ -1719,25 +1713,38 @@ values:
     name: CentralAfricanCFAFranc
     label: Central African CFA franc
     metadata:
-      symbol: "T"
-      precision: 2
-      fractional_unit: Sene
+      symbol: "FCFA"
+      precision: 0
       polity:
         - CMR
         - CAF
         - COG
         - TCD
         - GNQ
         - GAB
-      monetary_authority: Central Bank of Samoa
+      monetary_authority: Bank of the Central African States
+
+  - code: XAG
+    name: SilverTroyOunce
+    label: Silver troy ounce
+    metadata:
+      symbol: "Ag"
+      precision: 4
+  
+  - code: XAU
+    name: GoldTroyOunce
+    label: Gold troy ounce
+    metadata:
+      symbol: "Au"
+      precision: 4
```

Comment:
Do we want anybody to be able to create a “Silver troy ounce credit”, or “Gold troy ounce credit” instrument? Perhaps we need to add exceptions to exclude these from the currency selector for now (and the others commented below)?

Reply:
See my comments in Slack

Reply:
Resolution: should be added to this list for completeness. Can add separate filtering for CRN selection.